ydb/tests/tools/kqprun/configuration/app_config.conf

```
@@ -205,6 +205,33 @@ QueryServiceConfig {
       Value: "true"
     }
   }
+  
+  CheckpointsConfig {
+    Enabled: true
```

Наверное пока стоит отключить по умолчанию, судя по коду чекпоинты не заработают, если kqprun запустить без флага `-G 32000`

```suggestion
    Enabled: false
```